massa-execution-worker/src/slot_sequencer.rs

```diff
@@ -589,6 +596,7 @@ impl SlotSequencer {
         }
 
         // Check if the next candidate slot is available for execution.
+        #[cfg(not(feature = "slot-replayer"))]
```

Comment:
Same as before add a note explaining why we need to disable this for the slot replayer?

Reply:
done